form-view-model/src/main/kotlin/com/ritense/formviewmodel/error/MultipleFormException.kt

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ * Licensed under EUPL, Version 1.2 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.formviewmodel.error
+
+class MultipleFormException(
```

Comment:
This name kind of implies an exception about multiple forms. However, it's about one form having multiple errors.
Something in the line of `FormValidationException` would be better imho.